src/helix-ui/styles/core/hx-popover.less

```diff
@@ -10,8 +10,8 @@ hx-popover {
   font-weight: normal;
   left: 0;
   letter-spacing: normal;
-  max-height: 31.25rem; //500px
-  max-width: 31.25rem; //500px
+  max-height: 31.25rem; 500px
```

Comment:
Why were the comments removed making these sizes active? I don't know LESS all that well, but it looks like this would cause an error in the build...

Reply:
good catch, i removed some // comments throughout the file and those got caught!